real_time/surface_src/java_src/Alice/library/google-api-java-client/dependencies/css/print.css

```diff
@@ -0,0 +1,7 @@
+#banner, #footer, #leftcol, #breadcrumbs, .docs #toc, .docs .courtesylinks, #leftColumn, #navColumn {
+	display: none !important;

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1381948753/issues/source?bid=2938665&fileBranchId=2958290#l2)